events.js

```diff
@@ -0,0 +1,52 @@
+//*****VARIABLES*****
+
+var pageHeader = document.getElementById("page-header");
+var pageTitle = document.getElementById("page-title");
+var keypressInput = document.getElementById("keypress-input");
+var guineaPig = document.getElementById("guinea-pig");
+var addColor = document.getElementById("add-color");
+var makeLarge = document.getElementById("make-large");
+var addBorder = document.getElementById("add-border");
+var addRounding = document.getElementById("add-rounding");
+var outputTarget = document.getElementById("output-target");
+
+//*****FUNCTIONS*****
+
+document.addEventListener("click", function(e){
+	writeToOutputTarget(e.target.innerText);
+});
+
+pageTitle.addEventListener("mouseover", function(){
+	outputTarget.innerHTML = 'You moved your mouse over the header';
+});
+
+pageTitle.addEventListener("mouseout", function(){
+	outputTarget.innerHTML = 'You left me!!';
+});
+
+keypressInput.addEventListener("keypress", function(e){
```

Comment:
The output text seems to be one letter behind what the user has actually typed into the keypressInput field.  To fix this, you could change "keypress" to "keyup";  this would capture the input value after the user has finished typing the particular keystroke.